app/controllers/selections_controller.rb

```diff
@@ -1,18 +1,37 @@
+include SelectionsHelper
+
 class SelectionsController < ApplicationController
 
-  def add
-    @user = User.find(session[:user_id])
-    @station = Station.all
-    # figure this out
-    @selection = Selection.new(params:user_id, station_id)
+  def index
+    @selections = current_user.stations    #provides an array of station objects
```

Comment:
There should be only one space before and after `#`:
```@selections = current_user.stations # provides an array of station objects```

Reply:
Didn't realize that was a thing, will correct.